wcmc_components/app/controllers/wcmc_components/table_controller.rb

```diff
@@ -1,8 +1,8 @@
 module WcmcComponents
-  class TableController < ApplicationController
-    def index
-      @results = model_class.paginate_for_table(query_params_with_symbol_keys)
 
+  class TableController < WcmcComponents::ApplicationController
+    def list
```

Comment:
I think you might have worked off an older version at some point - the action name here should be index (it used to be called list). 

A tiinnyyy thing, there's a blank line on line 2 which we don't usually have. Not a big deal but nice to be consistent :)

Reply:
Oh yes! It must have happened when resolving the merge conflict.. Fixed now :)

Reply:
Ah yeah, I imagined as much. Cheers!